team_minutes/2020-07-24.md

```diff
@@ -221,7 +221,7 @@ R Consortium Diversity & Inclusion Group: Samantha Toet, gwynn sturdevant, Hadle
         - (CW) Excellent points, Robert. +1 from YB and NT.
         - (NT) Co-learning with peers can be better than learning from mentors. +1 from AM
         - (AM) I think it’s incorrect to say that someone who is not an expert is a worse mentor. Research shows that people closest to the novice state are often the best teachers.
-        - (GM) Sorry - I did not mean that one can only learn from an expert 
+        - (MG) Sorry - I did not mean that one can only learn from an expert 
```

Comment:
I don't think this was me...

Reply:
Could this `GM` also be `GB` as mentioned below?

Reply:
@MayaGans This was Robert Gentleman (RG), a comment copied out of chat.